### ISSUE TITLE
Request for CORS header

### ISSUE THREAD

**Issue by samatt** [alexandersimoes/oec#118]
<img width="1440" alt="screen shot 2016-01-15 at 6 27 57 pm" src="https://cloud.githubusercontent.com/assets/1969578/12368071/c546f488-bbb5-11e5-8495-b87e0cc04211.png">

**Comment by alexandersimoes** [comment]
Can't seem to recreate this... what was the exact API endpoint you were trying to access?

Here's a simple test page I made which seems to work (open the console when you load it).

http://alexandersimoes.com/crossdomaintest.html

**Comment by samatt** [comment]
Yeah, so ive not had a problem with the visual endpoint its only when I'm trying to get the raw data 
its this end point `http://atlas.media.mit.edu/hs07/import/2010.2012/usa/all/show`

if you want to see a live example go to `http://moreandmorestore.herokuapp.com/`

and look at the console.

In the JS there is the following line:
` $.ajax({type: "GET",url: "http://atlas.media.mit.edu/hs07/import/2010.2012/usa/all/show",}).done(function(data){ console.log(data);})`

**Comment by alexandersimoes** [comment]
I wonder if this is jQuery having some stricter issues around cross domain requests... the following worked fine for me in your console.

```JS
var oReq = new XMLHttpRequest();
oReq.open("GET", "http://atlas.media.mit.edu/hs07/import/2010.2012/usa/all/show/");
oReq.addEventListener("load", function(){ 
    console.log(JSON.parse(oReq.responseText));
});
oReq.send();
```
